apps/site/src/lib/components/ValidatingForm/ValidatingForm.tsx

```diff
@@ -11,12 +11,15 @@ interface FormProps {
 
 function ValidatingForm(props: PropsWithChildren<FormProps>) {
 	const [validated, setValidated] = useState<boolean>(false);
+	const [isClickable, setIsClickable] = useState<boolean>(true);
 
 	const handleSubmit = (event: FormEvent<HTMLFormElement>): void => {
 		const form = event.currentTarget;
 		if (!form.checkValidity()) {
 			// prevent submission to display validation feedback
 			event.preventDefault();
+		} else {
+			setIsClickable(false);
```

Comment:
Issue: when navigating back in the browser, the state is permanently disabled.

Reply:
I think this can also be resolved by the method I mentioned above. Also, I didn't run into this issue. Could this be a Safari thing?